lockfile/audit/src/types.ts

```diff
@@ -6,6 +6,13 @@ export interface AuditVulnerabilityCounts {
   critical: number
 }
 
+export interface IgnoreAuditVulnerabilityCounts {
```

Comment:
I think it should be
```suggestion
export interface IgnoredAuditVulnerabilityCounts {
```

Reply:
Done.